fbw-a380x/src/systems/systems-host/systems/LegacyGpws.ts

```diff
@@ -0,0 +1,814 @@
+// Note the master copy of these flags is contained in `fbw-a32nx\src\systems\shared\src\AutoCallOuts.ts`
+// Please do not edit here unless copying from there.
+
+import { Arinc429Word, NXDataStore } from '@flybywiresim/fbw-sdk';
+import { FmgcFlightPhase } from '@shared/flightphase';
+// eslint-disable-next-line camelcase
+import { LegacySoundManager, soundList } from 'systems-host/systems/LegacySoundManager';
+
+/** Bit flags for the radio auto call outs (for CONFIG_A32NX_FWC_RADIO_AUTO_CALL_OUT_PINS). */
+const RadioAutoCallOutFlags = Object.freeze({
```

Comment:
@Benjozork is moving the typescript copy of these to fbw-common from fbw-a32nx. This file should import those rather than redefining them.

Reply:
Good point. I'll then import the flags from fbw-a32nx in this PR, as long as no other PR moved it to fbw-common